### DISCUSSION TITLE
Does quartz.net support batch jobs like hangfire pro?

### DISCUSSION THREAD

**Discussion by bugproof** [quartznet/quartznet#1140] [Q&A] [answered]
https://www.hangfire.io/pro/#atomic-background-job-creation

Not gonna lie I like quartz.net license more compared to hangfire LGPL

**Comment by lahma** [comment, marked as answer]
No, you would need to write your own logic to handle that.

**Reply by bugproof** [reply]
@lahma I'm curious what do you think of hangfire as an author of this library? Are there things you like more in hangfire as compared to quartz.net?

**Reply by lahma** [reply]
I think it's a fine library with a good backing scheme via commercial options.

The libraries have different approaches and I personally find Quartz's job encapsulating logic easier to reason than capturing lambdas, but it's just a personal preference. 

Always evaluate between alternatives and choose what's best for your team.
